pkg/minikube/cluster/cluster.go

```diff
@@ -549,50 +549,54 @@ func CreateSSHShell(api libmachine.API, args []string) error {
 
 type ipPort struct {
 	IP   string
-	Port int
+	Port int32
 }
 
-func GetServiceURL(api libmachine.API, namespace, service string, t *template.Template) (string, error) {
+func GetServiceURLs(api libmachine.API, namespace, service string, t *template.Template) ([]string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
-		return "", errors.Wrap(err, "Error checking if api exist and loading it")
+		return nil, errors.Wrap(err, "Error checking if api exist and loading it")
 	}
 
 	ip, err := host.Driver.GetIP()
 	if err != nil {
-		return "", errors.Wrap(err, "Error getting ip from host")
+		return nil, errors.Wrap(err, "Error getting ip from host")
 	}
 
 	client, err := getKubernetesClient()
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	return getServiceURLWithClient(client, ip, namespace, service, t)
 }
 
-func getServiceURLWithClient(client *unversioned.Client, ip, namespace, service string, t *template.Template) (string, error) {
-	port, err := getServicePort(client, namespace, service)
+func getServiceURLWithClient(client *unversioned.Client, ip, namespace, service string, t *template.Template) ([]string, error) {
+	ports, err := getServicePorts(client, namespace, service)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
+	urls := []string{}
+	for _, port := range ports {
+		if t == nil {
```

Comment:
I do not believe that t can be nil with the current flow, I will remove this statement.